control/freqplot.py

```diff
@@ -380,26 +424,55 @@ def default_frequency_range(syslist):
     for sys in syslist:
         try:
             # Add new features to the list
-            features = np.concatenate((features, np.abs(sys.pole())))
-            features = np.concatenate((features, np.abs(sys.zero())))
+            if sys.isctime():
+                features_ = np.abs(sys.pole())
+                features_ = np.concatenate((features_, np.abs(sys.zero())))                
+                # Get rid of poles and zeros at the origin
+                features_ = features_[features_ != 0.0];
+                features = np.concatenate((features, features_))
```

Comment:
Would it make sense to remove all poles and zeros on (or very near) the imaginary axis?  It wouldn't make sense to remove such zeros for a Nyquist plot, I guess.